snscrape/modules/facebook.py

```diff
@@ -159,6 +165,11 @@ def _soup_to_items(self, soup, baseUrl, mode):
 
 class _FacebookUserAndCommunityScraper(_FacebookCommonScraper):
 	def __init__(self, username, **kwargs):
+		'''
+		Args:
```

Comment:
To match up with the rest of the docstrings, shouldn't there not be a newline there? (very minor i know but...)

Reply:
`Args`, `Kwargs`, `Returns`, `Yields`, etc. is usually positioned unindented in the left according to Google style. In addition to that, I feel uncomfortable putting `Args:` right after `'''` because it is usually reserved for docstring summary.

On the other hand, I don't think init docstring needs more explanation. The relevant information for each class is already given in the class docstring.